src/applications/discharge-wizard/components/v2/resultsComponents/ResultsSummary.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  determineBoardObj,
+  determineAirForceAFRBAPortal,
+  determineFormData,
+} from '../../../helpers';
+
+import {
+  SHORT_NAME_MAP,
+  RESPONSES,
+} from '../../../constants/question-data-map';
+
+const ResultsSummary = ({ formResponses }) => {
+  const forReconsideration =
+    [
+      RESPONSES.PREV_APPLICATION_TYPE_3A,
+      RESPONSES.PREV_APPLICATION_TYPE_3B,
+    ].includes(formResponses[SHORT_NAME_MAP.PREV_APPLICATION_TYPE]) &&
+    ![
+      RESPONSES.FAILURE_TO_EXHAUST_1A,
+      RESPONSES.FAILURE_TO_EXHAUST_1B,
+    ].includes(formResponses[SHORT_NAME_MAP.FAILURE_TO_EXHAUST]);
+
+  const airForceAFRBAPortal = determineAirForceAFRBAPortal(formResponses);
+
+  let summary = `Based on your answers, you need to complete Department of Defense (DoD) Form ${
```

Comment:
Consider: It'd be easier to grok what this text should be if we define the constants above, and insert them as variables into the string.